sdk/include/platform/sunburst/platform-rgbctrl.hh

```diff
@@ -0,0 +1,83 @@
+#pragma once
+#include <cdefs.h>
+#include <stdint.h>
+
+/**
+ * An enum representing each of the Sonata's RGB LEDs.
+ */
+enum class SonataRgbLed
+{
+	Led0,
+	Led1,
+};
+
+/**
+ * A driver for the Sonata's RGB LED Controller
+ */
+struct SonataRgbLedController
+{
+	/**
+	 * Registers for setting the 8-bit red, green, and blue values
+	 * for the two RGB Leds.
+	 */
+	uint32_t rgbLed0;
+	uint32_t rgbLed1;
+	/// Control Register
```

Comment:
It would be nice to document the bits here.  You've done that properly with the enums below, so just a forward ref saying that these are set with the values from the control enum below would be nice.